### PR TITLE
Define remote environments in the props.json file 

### PR DESCRIPTION
**Pull request by javifernandez**
In FirefoxReality there was the possibility of retrieving environments from the github page of a repository. We use the props.json file we can now define for each Wolvic's version a structure with the following information of the environments we offer to the user:

  {
        "value": "spring",
        "title": "Spring",
        "thumbnail": "https://igalia.github.io/wolvic-3D-environments/spring/spring.png",
        "payload": "https://igalia.github.io/wolvic-3D-environments/spring/spring.zip"
  }

The dialog we have in the Wolvic's settings will show all options for all the entries defined in the corresponding version, downloading just the thumbnail. Once the user select an option, a download tasks is queued and only once it finished, the new env will be loaded.

### CODE REVIEW

**Review by svillar — CHANGES_REQUESTED** [review]
So I don't completely understand the purpose of this PR. We already distribute that environment with Wolvic. Are you proposing not to do that and let users download it?

If that's the case then we should also remove the assets from the source tree.

Also you're adding a zip file with a filename which is not in the payload of the json file.

**Comment by javifernandez** [comment]
> So I don't completely understand the purpose of this PR. We already distribute that environment with Wolvic. Are you proposing not to do that and let users download it?

Yes, we need at least one env to test the "feature"

> 
> If that's the case then we should also remove the assets from the source tree.

Sure, but that has to be done in a different PR, since this goes to the gh-pages branch. 

> 
> Also you're adding a zip file with a filename which is not in the payload of the json file.

This is handled in the PR#634, where we add some logic to add the _srgb suffix to the file name.

**Comment by svillar** [comment]
> > So I don't completely understand the purpose of this PR. We already distribute that environment with Wolvic. Are you proposing not to do that and let users download it?
> 
> Yes, we need at least one env to test the "feature"

Indeed I understand that it's good to test it, but once you have tested that it works fine, why should we add an environment that we already distribute? I don't see the point honestly

**Comment by javifernandez** [comment]
As we discussed, we have a dedicated github repo for the remote environments, so this PR just updates the props.json to add new entries for the environments.

**Comment by svillar** [comment]
The new environment does not work on devices not supporting srgb

Apart from that we should use another name, like Wolvic 1.4 or something

**Comment by bkardell** [comment]
Can I ask: What device doesn't support srgb? The entire web is srgb, what happens there?

**Comment by svillar** [comment]
> Can I ask: What device doesn't support srgb? The entire web is srgb, what happens there?

It isn't about the device, so badly expressed from my side. This has to do with the GL formats supported by the runtime on each device. And several of them definitely do not support textures in SRGB format.

**Review by svillar — CHANGES_REQUESTED** [review]
I've just realized that I forgot to push the 1.4 entry when I released it. I had the commit locally. I've just pushed it.

Please do change this entry for 1.4.1

File: props.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
 {
+  "1.4": {
+    "whatsNewUrl": "https://wolvic.com/blog/release_1.4",
+    "environments": [
+      {
+        "value": "skybox_r90",
+        "title": "Skybox R90",
```
Comment (svillar) [inline props.json:7]: Please use another name as suggested in some other PR. We can call it "Flinstones" or Wolvic 1.4 if you don't want to be supercreative :)

Comment (javifernandez) [reply props.json]: Done

**Comment by svillar** [comment]
> > Can I ask: What device doesn't support srgb? The entire web is srgb, what happens there?
> 
> It isn't about the device, so badly expressed from my side. This has to do with the GL formats supported by the runtime on each device. And several of them definitely do not support textures in SRGB format.

Again sorry, thanks to @javifernandez I realized that when I mentioned sRGB what I really meant was KTX. So indeed Pico cube layer does not support rendering compressed ETC2 textures. We use PNGs for Pico (which actually are sRGB BTW)

**Review by svillar — CHANGES_REQUESTED** [review]
Mind updating this? I think we should aim for 1.5.1 here

**Review by svillar — APPROVED** [review]
Finally!

**Review by svillar — CHANGES_REQUESTED** [review]
Well no!!! I think it's better to add it in 1.5.1 otherwise it'd be weird to get a new environment with a OTA-like update

**Comment by javifernandez** [comment]
> Well no!!! I think it's better to add it in 1.5.1 otherwise it'd be weird to get a new environment with a OTA-like update

It'd be better to wait then until we create the release notes for the 1.5.1.

**Review by svillar — APPROVED** [review]

**Pull request merged**